README.md

```diff
@@ -88,18 +66,15 @@ After you're finished please remove all the comments and instructions!
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
-
-> Describe between 1-3 key features of the application.
 - added index.HTML
 - added Style.CSS
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
+# LIVE DEMO 
 
-- comming soon
+- [] comming soon
 
-> Add a link to your deployed project.
 
 - [Live Demo Link](https://google.com)
```

Comment:
- [ ] [OPTIONAL] Kindly discard the live demo since, the project doesn't and will likely not have a live demo. 😃